tests/unit/MyParks.spec.js

```diff
@@ -0,0 +1,85 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import MyParks from '@/components/MyParks.vue';
+import Vuex from 'vuex';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('MyParks.vue', () => {
+	let state;
+	let store;
+	beforeEach(() => {
+		state = {
+			savedParks: [],
+		};
+
+		store = new Vuex.Store({
+			state,
+		});
+	});
+
```

Comment:
Nice to see how you implemented Vuex into the test. I'm sure I'll be looking at this later tonight when I do more testing on my branch.